templates/notifyPopup.hbs

```diff
@@ -26,14 +26,50 @@
             </div>
           </div>
         {{/if}}
-
-        {{#if body}}
-        <div class="notify__body">
-          <div class="notify__body-inner">
-            {{{compile body}}}
+        
+        <div class="notify__section">
+          <div class="notify__section-inner">
+          
+            {{#if _graphic._src}}
+            {{#any (equals _imageAlignment 'left') (equals _imageAlignment 'top')}}
+            <div class="notify__image-container{{#if _graphic.attribution}} has-attribution{{/if}}">
+              <img class="notify__image" src="{{_graphic._src}}" {{#if _graphic.alt}}aria-label="{{{_graphic.alt}}}"{{else}}aria-hidden="true"{{/if}}>
+              {{#if _graphic.attribution}}
+              <div class="notify__attribution">
+                <div class="notify__attribution-inner">
+                  {{{_graphic.attribution}}}
+                </div>
+              </div>
+              {{/if}}
+            </div>
+            {{~/any~}}
+            {{~/if~}}
```

Comment:
Minor: I don't think this tilde is consistent with the opening statement any more. Can you update that too please?

Reply:
I'm not entirely sure how the tildes work if i'm honest. I thought they just removed whitespace if you have two encasing a region that would otherwise have whitespace?

Reply:
Actually, I think it is fine. I had been thrown because an example I had seen seemed to indicate the tilde was required on opening and closing of the `if`, but I can see now that it's actually more complex than that. Happy for you to ignore 👍

Reply:
Ta mate